index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Intuit account</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    <!--------------------------Header--------------------->
+    <header>
+        <div class="header-text">
+            <h5>Already have an intuit account?</h5>
+            <a class="header-link" href="/blank"><p>I forgotmy User ID or password</p></a>
+        </div>
+        <div class="header-btn-container">
+            <button class="signin-btn"><img class="btn-icon" src="./graphics/icons8-lock-26.png" alt="lock">Sign In</button>
+        </div>
+    </header>
+    <!------------------------------Main------------------>
+    <main>
+        <div>
+            <div class="signup-title">
+                <h1>Intuit</h1>
+                <br>
+                <div class="brand-links">
+                    <a id="brand" class="icons" href="https://www.mint.com/"><img class="icons" src="./graphics/uokpl.rs-mint-logo-png-6076604.png" alt="icon"><h3>mint</h3></a>
+                    <a id="brand" class="icons" href="https://quickbooks.intuit.com/"><img class="icons" src="./graphics/icons8-quickbooks-64.png" alt="icon"><h3>quickbooks</h3></a>
+                    <a id="brand" class="icons" href="https://turbotax.intuit.com/"><img class="icons" src="./graphics/turbotax.png" alt="icon"><h3>turbotax</h3></a>
+                </div>
+            </div>
+            <div class="form-section">
+                <section class="form-title">
+                    <h2>Creat an Intuit account</h2>
+                </section>
+                    <div class="learn-more">
+                        <h4>One account for everything Intuit. including Mint.<a href="https://accounts-help.lc.intuit.com/questions/1582580-creating-an-account"><h5>Learn more</h5></a></h4>
```

Comment:
it will be nice if the learn more text was having a different color so the user can notice it as a link.